lib/govuk_publishing_components/presenters/checkboxes_helper.rb

```diff
@@ -16,7 +16,8 @@ class CheckboxesHelper
                   :description,
                   :heading_size,
```

Comment:
If `heading_size` is only being used to capture a passed option and `@size` is replacing it in the code, I don't think it needs to be in the `attr_reader` list anymore.

Reply:
We use this in the size method now, should we still remove it?

Reply:
Yes, I think so... prefixing it with the `@` makes it a class level variable, so it can be accessed anywhere in the class (as you're doing). Adding it to the `attr_reader` list means it can be accessed from outside the class, which I don't think you're doing anymore? So I think it can be removed, give it a try.

Expert ruby devs, please be gentle if I've said any of the above wrong...

Reply:
I've removed it...

tests pass so I'm merging